src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -68,10 +64,36 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
                     "-${ShuttleCliOptions.CREATE}=$email"))
         }
 
+        /**
+         * Import datasets via Shuttle
+         * @param
+         */
+        fun importAtlasDataSet(flightFileName: String, flightConfigurationName: String, flightConfigurationSource: String) {
```

Comment:
Are we always going to use the same config file? If so, we could make the two imort functions more general, with providing like: 
```
fun importDataSet(flightFileName: String, dataSourceName: String, loadFromFile: Boolean)
```